cineast-api/src/main/java/org/vitrivr/cineast/api/websocket/handlers/queries/TemporalQueryMessageHandler.java

```diff
@@ -10,6 +9,8 @@
 import java.util.concurrent.CompletableFuture;
 import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+
+import it.unimi.dsi.fastutil.ints.Int2ObjectLinkedOpenHashMap;
```

Comment:
FastUtil was imported via ADAMpro and was used in some places, despite Cineast traditionally using Trove for such purposes. They both do very similar things. I don't know which one is better, I've mostly worked with Trove in the past. In the context of such a cleanup effort, I think it would be worth to just use one of the two.

Reply:
I've removed fastutil again and replaced the two usages with their trove-equivalent